integrations/tensorflow/.bazelignore

```diff
@@ -14,3 +14,6 @@
 
 # Ignore build files that are intended for other Bazel repositories
 build_tools/overlay
+
+# Ignore build files open-sourced only for reference and usage by Bazel->CMake
+e2e/
```

Comment:
we may need to exclude this directory from buildifier also

Reply:
https://github.com/google/iree/blob/02bb838cf985917ab04457d7e699a59deadc8515/.github/workflows/lint.yml#L38-L39

Reply:
@GMNGeoffrey lint error logs: https://github.com/google/iree/pull/5117/checks?check_run_id=2117047415 (this is from https://github.com/google/iree/commit/4df7b58941658f810564cdd4b9b5eb85992337a9)

Reply:
Argh sorry I missed this :-( The approach I took at https://github.com/GMNGeoffrey/iree/blob/ba218e24027478235711bb9791d6d992c6a37c76/.github/workflows/lint.yml#L44 was to disable the native-py check (which we want to disable anyway). I think the latest version of buildifier doesn't even have it because the future of native rules is not really certain: https://github.com/bazelbuild/buildtools/issues/923. I'm actually a bit skeptical we're getting much value out of this actionification of buildifier vs just wgetting and running the binary ourselves.